demo/src/app/case-create-filter.component..ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'case-create-filter',
+  template: `<div class="container-fluid">
+                <ccd-create-case-filters></ccd-create-case-filters>
+             </div>`
+})
+export class CaseCreateFilterComponent {
```

Comment:
Can we rename to CaseCreateFilterConsumerComponent here to make it consistent with the others?

Reply:
Done